config/locales/en.yml

```diff
@@ -657,70 +657,77 @@ en:
   terms:
     body_html: |
       <h2>Privacy Policy</h2>
-
       <h3 id="collect">What information do we collect?</h3>
 
-      <p>We collect information from you when you register on our site and gather data when you participate in the forum by reading, writing, and evaluating the content shared here.</p>
-
-      <p>When registering on our site, you may be asked to enter your name and e-mail address. You may, however, visit our site without registering. Your e-mail address will be verified by an email containing a unique link. If that link is visited, we know that you control the e-mail address.</p>
+      <ul>
+        <li><em>Basic account information</em>: If you register on this server, you may be asked to enter a username, an e-mail address and a password. You may also enter additional profile information such as a display name and biography, and upload a profile picture and header image. The username, display name, biography, profile picture and header image are always listed publicly.</li>
+        <li><em>Posts, following and other public information</em>: The list of people you follow is listed publicly, the same is true for your followers. When you submit a message, the date and time is stored as well as the application you submitted the message from. Messages may contain media attachments, such as pictures and videos. Public and unlisted posts are available publicly. When you feature a post on your profile, that is also publicly available information. Your posts are delivered to your followers, in some cases it means they are delivered to different servers and copies are stored there. When you delete posts, this is likewise delivered to your followers. The action of reblogging or favouriting another post is always public.</li>
```

Comment:
The fact that we list here explicitly all the different types of data you can create makes me a little scared that we'll have to update the privacy policy every time we add a new feature, which would be bad. For example, where do reports fall under here? What if we made lists public? You might want to add a more general category, or something like "information we collect from you includes but is not limited to"

While the specificity is good and useful, because you removed the very general `We collect information from you when you register on our site and gather data when you participate in the forum by reading, writing, and evaluating the content shared here` it's possible someone could read this and conclude that recording who made a report is against our privacy policy, or that displaying a list will be, which would be bad.

Reply:
I think the idea that a privacy policy is something you set and forget is antiquated and problematic.  As convenient as it might be in the short term to just not bother with it, we developers have to come to grips with the fact that the privacy policy is something that grows with the site, just like test suites and documentation. Point of fact, the privacy policy IS documentation, and it needs to remain both up to date, and specific.  Developers _should_ make a habit of updating the privacy policy whenever the data collected changes.  (ideally, this could be at least partially automated)

"Includes but is not limited to" effectively expands the bounds on what could be collected to infinity, which kind of defeats the purpose of this document.

Reply:
I am inclined to agree that being explicit, and updating when we need to collect more, is what makes a privacy policy useful and trustworthy...